### PR TITLE
Project Setup

### PR DESCRIPTION
**Pull request by PeterChav01**
## Checklist

- Project with UIKit created
- Project with SwiftUI created
- `.gitignore` file generated

### CODE REVIEW

File: ML App SwiftUI/ML App SwiftUI/ViewController.swift [hunk 1]
```diff
@@ -0,0 +1,19 @@
+//
+//  ViewController.swift
+//  ML App SwiftUI
+//
+//  Created by Jose Chavez Aparicio on 20/10/22.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
```
Comment (AlejandroVC67) [inline ML App SwiftUI/ML App SwiftUI/ViewController.swift]: We don't need a UIVIewController for the SwiftUI side
Also, rename the folder, it is better to have no spaces between words, so just call it SwiftUI

File: ML App UIKit/ML App UIKit/ViewController.swift [hunk 2]
```diff
@@ -0,0 +1,19 @@
+//
+//  ViewController.swift
+//  ML App UIKit
+//
+//  Created by Jose Chavez Aparicio on 20/10/22.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
```
Comment (AlejandroVC67) [inline ML App UIKit/ML App UIKit/ViewController.swift]: Same comment about the extra spaces for the folder name, you can just call it UIKit

**Review by AlejandroVC67 — APPROVED** [review]

File: ML App SwiftUI/ML App SwiftUI/ViewController.swift (hunk 1 above)
Comment (PeterChav01) [reply ML App SwiftUI/ML App SwiftUI/ViewController.swift]: Controller deleted and created SwiftUIview
Folder name updated

File: ML App UIKit/ML App UIKit/ViewController.swift (hunk 2 above)
Comment (PeterChav01) [reply ML App UIKit/ML App UIKit/ViewController.swift]: Folder name updated

**Pull request merged**